README.md

```diff
@@ -1,11 +1,55 @@
-NumFOCUS Elections
-==================
+# NumFOCUS Board Elections
 
-This repository holds issues and documents related to the running of elections
-of the NumFOCUS Board.
+NumFOCUS has a Board of Directors who provide guidance, oversight, and support for various NumFOCUS-led initiatives.[^1]
+Board members have fixed term limits. As members complete their term or step-down, new members need to be elected to the board. Historically, there has not been a formal process for recruiting and admitting new board members.
 
-We are using [issues](https://github.com/numfocus/elections/issues) to discuss
-and track work items.
+A [volunteer-led Election Committee](election-committee.md) has been formed to plan and execute board elections in a fair, unbiased, and transparent manner.
 
-[Previous meeting minutes can be found
-here](https://docs.google.com/document/d/1zEYZfbOug2xMzvKTQ-1txtrgj-5-FOslle7Dd2RBzOM/edit#heading=h.lb4hte78frnp).
+[^1]: [Who is the board & how does the board serve NumFOCUS?](https://numfocus.medium.com/who-is-the-board-how-does-the-board-serve-numfocus-b109d0c0dd17)
+
+> [!NOTE]
+> This is the first NumFOCUS board election and the election committee is a newly formed team.
+> We’ll create and document the procedural details in this repository as we conduct the election.
+> You can track ongoing activities on the Issue Tracker, and share your thoughts. We’d love input from the community in all aspects of this election.
+
+## Upcoming election 🗳️
+
+### Open seats (needs conformation)
+
+* Technical Executive Committee - 5 positions
+* Administrative Board - 3 positions
+
+<!-- TODO: One sentence overview of responsibilities for Technical Executive Committee and Administrative Board. This is a new change that the community won't be familiar with. -->
+
+### Process and timeline
+
+The election process will involve the following board phases. The dates are tentative and are subject to change depending on the community engagement during nomination and voting phases.
+
+| Election phases | Tentative dates |
+| - | - |
+| Nomination period | May 20th - June 3rd |
+| Nominee application review (by election committee and NumFOCUS board) | June 3rd - June 17th |
+| Notify nominees and get confirmation | June 17th - June 24th |
+| Voting period| June 24th - July 8th |
+| Review election results (by  NumFOCUS board) | July 8th  - July 15th |
+| Publish results announcing new board members | mid-end July |
+
+## Contents 🗂️
+
+This repository hosts documents related to NUmFOCUS board elections, and ongoing tasks & activities are tracked on the [issue tracker](https://github.com/numfocus/elections/issues).
+
+* [Election process](election-process.md) (coming soon)
+* [Board member responsibilities](board-responsibilities.md) (coming soon)
+* [Election committee details](election-committee.md)
+* [Frequently asked questions](faqs.md)
+
+## Reference documents 📑
+
+<!-- TODO: Add links -->
+
+* NumFOCUS Bylaws
+* Election Committee Charter
```

Comment:
Have we uploaded the charter somewhere? If not, shall we create a PDF and add it to this repo?

Reply:
added a todo, this doesn't need to be a blocker